schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals ( 
+    id              INT GENERATED ALWAYS AS IDENTITY,
+    name            VARCHAR(250),
+    date_of_birth   DATE, 
+    escape_attempts INT,
+    neutered        BOOLEAN, 
+    weight_kg       DECIMAL,
+    PRIMARY KEY(id) ;
```

Comment:
- [ ] Kindly remove the semi-colon after the `PRIMARY KEY(ID)` line of code. It creates a **_syntax error_** whenever I try to create the `animals` table. 💻 😎 


![image](https://user-images.githubusercontent.com/77106962/228659987-c4220240-dc08-4793-a772-b102bee28b30.png)

Reply:
Thank you for the review and good feedback, I have implemented the required changes, kindly review it again @Gambit142